packages/suite-desktop/src-electron/libs/menu.ts

```diff
@@ -38,64 +27,86 @@ const mainMenuTemplate: MenuItem[] = [
             { role: 'cut' },
             { role: 'copy' },
             { role: 'paste' },
-            ...(isMac
-                ? [
-                      { role: 'pasteAndMatchStyle' },
-                      { role: 'delete' },
-                      { role: 'selectAll' },
-                      { type: 'separator' },
-                      {
-                          label: 'Speech',
-                          submenu: [{ role: 'startspeaking' }, { role: 'stopspeaking' }],
-                      },
-                  ]
-                : [{ role: 'delete' }, { type: 'separator' }, { role: 'selectAll' }]),
+            // extended below
         ],
     },
     // { role: 'viewMenu' }
     {
         label: 'View',
         submenu: [
             { role: 'reload' },
-            { role: 'forcereload' },
-            { role: 'toggledevtools' },
+            { role: 'forceReload' },
+            { role: 'toggleDevTools' },
```

Comment:
would be nice to hide it for production builds, as it is working only in dev mode...

Reply:
addressed 80ee802